editors/code/src/main.ts

```diff
@@ -167,6 +167,7 @@ async function bootstrapExtension(config: Config, state: PersistentState): Promi
         }
         return;
     };
+    if (serverPath(config) !== null) return;
```

Comment:
Oh what a joy: according to my debugging, this now returns `undefined` despite its signature and `undefined !== null` is `true` ergo https://rust-lang.zulipchat.com/#narrow/stream/185405-t-compiler.2Fwg-rls-2.2E0/topic/Nightlies.20stopped.3F/near/222700457

Reply:
I'll try to debug more to be 100% sure and somehow adkust the code, but I'm terruble with TS, so not sure when I'm finishing this.